src/Analyzers/CSharp/CodeFixes/UsePatternCombinators/CSharpUsePatternCombinatorsCodeFixProvider.cs

```diff
@@ -148,7 +148,9 @@ public MyCodeAction(string title, Func<CancellationToken, Task<Document>> create
             {
             }
 
+#if !CODE_STYLE // 'CodeActionPriority' is not a public API, hence not supported in CodeStyle layer.
             internal override CodeActionPriority Priority => CodeActionPriority.Low;
+#endif
```

Comment:
how does this work in vs @mavasani ?  do we pick up the CodeStyle fixer? Or do we pick up the one from Features?

Reply:
We prefer the fixer and analyzer from NuGet package over the one from VSIX

Reply:
Btw is there a specific reason for marking this as low priority?

Reply:
@mavasani From related discussion in https://github.com/dotnet/roslyn/pull/43596, it seems this was a copy from another feature

![image](https://user-images.githubusercontent.com/31348972/160347922-51e8ac81-1c0e-4c21-8315-b70ec9339bfe.png)

I think if overriding `Priority` is something important the API may be considered to be exposed publicly

Reply:
cc @alrz